collector/mesos_agent/mesos_agent.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package mesos_agent
```

Comment:
Why are we renaming this package? It's common convention in golang to underscore, not camelcase, the package name: https://blog.golang.org/package-names